Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -23,6 +23,8 @@ public class LabelRenderer : ViewRenderer<Label, TextView>
 		FormsTextView _view;
 		bool _wasFormatted;
 
+		readonly MotionEventHelper _motionEventHelper = new MotionEventHelper();
```

Comment:
Can we remove readonly and set this to null on Dispose ? Just worry we could leak by setting a element reference on MotionEventHelper